pkg/apis/kubermatic/v1/external_cluster.go

```diff
@@ -74,8 +74,42 @@ type ExternalClusterCloudSpec struct {
 	KubeOne *ExternalClusterKubeOneCloudSpec `json:"kubeone,omitempty"`
 }
 
+type KubeOneExternalClusterState string
+
+const (
+	// PROVISIONING state indicates the cluster is being imported.
+	PROVISIONING KubeOneExternalClusterState = "PROVISIONING"
```

Comment:
Please use regular constant names like "Provisioning". We might even want some suffix or prefix to make this set of constant more distinct, like `ProvisioningStatus` or `StatusProvisioning` maybe.

Reply:
Also, where are these constants coming from? Are we defining them ourselves? If so, can we make them non-screaming? I presume that we want to show them when admins to a `kubectl get externalclusters` and HAVING THE LIST SCREAM AT THE USER IS SUPER RUDE AND ALSO UPPERCASE IS HARD TO READ AT LEAST TO ME.

Reply:
For comparision, our project phases are also "Active", "Inactive" and "Terminating", not uppercased. :)

Reply:
agreed